jOOQ/src/main/java/org/jooq/impl/DSL.java

```diff
@@ -7858,6 +7860,22 @@ public static DropIndexOnStep dropIndexIfExists(Index index) {
     // XXX Access control
     // -------------------------------------------------------------------------
 
+    //todo add javadoc
+    public static Grant<Record> grant(String privilege) {
+        return using(new DefaultConfiguration()).grant(new PrivilegeImpl(name(privilege)));
+    }
+
+    //todo add javadoc
+    public static Grant<Record> grant(List<String> privileges) {
```

Comment:
While the convenience method `grant(String privilege)` is fine, this method is not. If we have a `Privilege` type, then the collection overload must either:

- Accept the `Privilege` type through `Collection<? extends Privilege>`
- Accept any type through `Collection<?>`

I prefer the former. Note also the use of `Collection` rather than `List` in the API.

Reply:
... note, it is correct in `DSLContext`